src/YoloDev.Expecto.TestSdk/settings.fs

```diff
@@ -14,58 +18,159 @@ type RunSettings =
     disableParallelization: bool
 
     /// Gets a value which indicates the target framework the tests are being run in.
-    targetFrameworkVersion: string option }
+    targetFrameworkVersion: string option 
+
+    /// Gets the [ExpectoConfig](https://github.com/haf/expecto#the-config) that was set via RunSettings
+    expectoConfig : ExpectoConfig}
 
 [<RequireQualifiedAccess>]
 module RunSettings = 
   let defaultSettings = 
     { collectSourceInformation = true
       designMode = true
       disableParallelization = false
-      targetFrameworkVersion = None }
-  
-  let read (runSettings: IRunSettings) = 
-    let settings = defaultSettings
+      targetFrameworkVersion = None
+      expectoConfig = ExpectoConfig.defaultConfig }
+
+  let private foldCLIArgumentToConfig = function
+    | Sequenced -> fun o -> { o with ``parallel`` = false }
+    | Parallel -> fun o -> { o with ``parallel`` = true }
+    | Parallel_Workers n -> fun o -> { o with parallelWorkers = n }
+    | Stress_Timeout n -> fun o -> { o with stressTimeout = TimeSpan.FromMinutes n }
+    | Stress_Memory_Limit n -> fun o -> { o with stressMemoryLimit = n }
+    | Fail_On_Focused_Tests -> fun o -> { o with failOnFocusedTests = true }
+    | Debug -> fun o -> { o with verbosity = Expecto.Logging.LogLevel.Debug }
+    | Log_Name name -> fun o -> { o with logName = Some name }
+    | Filter hiera -> fun o -> {o with filter = Test.filter (fun s -> s.StartsWith hiera )}
+    | Run tests -> fun o -> {o with filter = Test.filter (fun s -> tests |> List.exists ((=) s) )}
+    | FsCheck_Max_Tests n -> fun o -> {o with fsCheckMaxTests = n }
+    | FsCheck_Start_Size n -> fun o -> {o with fsCheckStartSize = n }
+    | FsCheck_End_Size n -> fun o -> {o with fsCheckEndSize = Some n }
+    | Allow_Duplicate_Names -> fun o -> { o with allowDuplicateNames = true }
+    | No_Spinner -> fun o -> { o with noSpinner = true }
+    | _ -> id
```

Comment:
What's the fallback here for?

Reply:
CLIArguments that we can't really implement easily. I could make them explicit.